### DISCUSSION TITLE
nuxt的入口文件在哪里，如何设置封装axios能拦截所有请求？

### DISCUSSION THREAD

**Discussion by rehtafuoy** [nuxt/nuxt#18797] [Questions]
最近开发的时候，遇到一个逻辑问题。
项目在静态文件夹下的js文件夹中定义了个common.js文件，里面封装了api.js、request、axios之类的常规的东西。但是我在项目中，单页里面不想使用统一封装的，所以在该单页里面直接import axios from 'axios'引入了axios对象，然后axios.post(url，param)的方式发送请求。
但是结果还是被common.js中封装的公共req、res拦截器拦截了，而且依然没有逃过api.js的拦截（该文件会在所有请求url前面添加/api，以便让porxy进行处理）。
按照开发语言的惯例，饮用了对象，才会被对象内部的函数进行处理。我调用import axios from 'axios'使用的应该是公共plus插件，没用调用api.js、request之类的，为什么还会被拦截处理呢？我能想到的就是项目本身进行了全局封装，但是找了很久没有找到。所以想请教下，nuxt里面的入口文件如何定义的，在哪些地方设置可以影响全局

**Comment by M-Sayeh** [comment]
Fack you mother
